src/explainer.ts

```diff
@@ -1,30 +1,37 @@
 'use strict';
 
-import k8s = require('k8s');
+import request = require('request');
+import * as kubernetes from '@kubernetes/client-node';
 import * as pluralize from 'pluralize';
-import * as kubeconfig from './kubeconfig';
 import { formatComplex, formatOne, Typed, formatType } from "./schema-formatting";
 
 export function readSwagger(): Promise<any> {
-    return kubeconfig.readKubectlConfig().then((kc) => readSwaggerCore(kc));
-}
+    const kc = new kubernetes.KubeConfig();
+    kc.loadFromDefault();
```

Comment:
Does this behave correctly if the user has overridden the default `kubeconfig` path in their settings?

Reply:
The previous version didn't, so it's not a regression (and never mattered much because we were only getting the Swagger, not performing cluster operations), but it would be good not to propagate the error.

Reply:
Fixed.